lib/rules/web/admin_console/4.4/overview.xyaml

```diff
@@ -6,11 +6,12 @@ wait_overview_items_loaded:
 check_list_heading_shown:
   element:
     selector:
-      xpath: //button[contains(@class,'project-overview__item-heading--name') and text()='<heading>']
+      xpath: //button[contains(@class,'project-overview__item-heading--name') and text()='<heading>' and not(contains(@class,'pf-m-disabled'))]
```

Comment:
Why adding `pf-m-disabled` ?

Reply:
Just for incase the button is disabled for tmp, will remove if it's not needed.